src/js/helpers.js

```diff
@@ -0,0 +1,8 @@
+export const toMoney = (value, precision = 1, showUnit = true) => {
+  const unit = showUnit ? "$" : "";
+  return value === 0 ? "0" : `${unit}${(value / 1000).toFixed(precision)}K`;
```

Comment:
Nitpick: Why not show a unit for 0 if showUnit is true?

Not a blocker since I don't think we have any money values w/ 0